internal/compiler/typeloader.rs

```diff
@@ -840,7 +840,7 @@ impl Snapshotter {
 pub struct TypeLoader {
     pub global_type_registry: Rc<RefCell<TypeRegister>>,
     pub compiler_config: CompilerConfiguration,
-    style: String,
+    pub resolved_style: String,
```

Comment:
I would have left the name as is. But can't hurt.
Maybe a doc comment explaining what is is exactly would be good.